cosmo_tester/test_suites/test_snapshots/snapshots_multiple_managers_test.py

```diff
@@ -0,0 +1,176 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+import os
+import random
+import shutil
+import string
+import tempfile
+
+from cloudify_rest_client.exceptions import CloudifyClientError
+
+from cosmo_tester.framework.cfy_helper import CfyHelper
+from cosmo_tester.framework.testenv import bootstrap, teardown
+from cosmo_tester.framework.util import create_rest_client, YamlPatcher
+from cosmo_tester.test_suites.test_blueprints.nodecellar_test import (
+    OpenStackNodeCellarTest)
+
+
+def setUp():
+    bootstrap()
+
+
+def tearDown():
+    teardown()
+
+
+class TwoManagersTest(OpenStackNodeCellarTest):
+    """
+    This test bootstraps managers, installs nodecellar using the first manager,
+    checks whether it has been installed correctly, creates a snapshot,
+    downloads it, uploads it to the second manager, uninstalls nodecellar using
+    the second manager, checks whether nodecellar is not running indeed and
+    tears down those managers.
+    """
+
+    def setUp(self):
+        super(TwoManagersTest, self).setUp()
+        configuration = self.env.handler_configuration
+        self.workdir2 = tempfile.mkdtemp(prefix='cloudify-testenv-')
+
+        if 'second_manager_ip' in self.env.handler_configuration:
```

Comment:
Hi,
Seems like this code is not relevant... [second_manager_ip in handler_configuration]